### PR TITLE
Update zsh packages to 5.6.1

### PR DESCRIPTION
**Pull request by agross**
_Motivation:_ zsh 5.6.1 has been released

### Checklist
- [x] Build rule `all-supported` completed successfully
- [x] Package upgrade completed successfully
- [x] New installation of package completed successfully

### CODE REVIEW

File: spk/zsh-static/Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 SPK_NAME = zsh-static
 SPK_CONFLICT = zsh
-SPK_REV = 2
+SPK_REV = 1
```
Comment (ymartin59) [inline spk/zsh-static/Makefile]: Revision must be 3

File: spk/zsh/Makefile [hunk 2]
```diff
@@ -1,10 +1,10 @@
 # Allow overriding the name, used by spk/zsh-static.
 SPK_NAME ?= zsh
-SPK_VERS = 5.5.1
+SPK_VERS = 5.6.1
 SPK_CONFLICT ?= zsh-static
 # Allow overriding the revision, used by spk/zsh-static.
 # If you rev here, please also rev spk/zsh-static.
-SPK_REV ?= 6
+SPK_REV ?= 1
```
Comment (ymartin59) [inline spk/zsh/Makefile]: Revision must be 7

**Comment by agross** [comment]
OK, I thought it would be revision per package version, i.e. this is the first revision of the zsh-5.6.1 package.

**Comment by ymartin59** [comment]
Published

**Comment by agross** [comment]
Thanks. So SPK_REV is an ever-increasing number. Did I understand that
correctly?

Alex
--
http://therightstuff.de
Tiny phone, tiny mail

**Pull request merged**